### DISCUSSION TITLE
应用升级的时候没保留挂载目录

### DISCUSSION THREAD

**Discussion by seth-shi** [1Panel-dev/1Panel#7050] [Q&A]
### 1Panel 版本

v1.10.20-lts

### 问题描述

应用升级的时候没保留挂载目录

### 重现步骤

应用升级的时候没保留挂载目录

**Comment by wanghe-fit2cloud** [comment]
应用备份时仅包含应用的安装目录，建议将持久化目录保持为默认的 ./data。
若持久化目录修改至安装目录之外的路径，将不会纳入备份范围的。
